integration/js/BackgroundReplacementTest.js

```diff
@@ -0,0 +1,29 @@
+const {ClickBackgroundReplacementButton} = require('./steps');
+const {VideoBackgroundReplacementCheck} = require('./checks');
+const {TestUtils} = require('kite-common');
+const BackgroundFilterBaseTest = require('./utils/BackgroundFilterBaseTest');
+
+class BackgroundReplacementTest extends BackgroundFilterBaseTest {
+  constructor(name, kiteConfig) {
```

Comment:
I think you still need to pass the test name here and for the blur test as well. We use test name to emit test metrics in CW.